traits.yml

```diff
@@ -306,3 +831,30 @@ specifications:
             name: Stable
         usage:
           - relationship
+  representation:
+    description: >
+      Specifications for alternative presentations of a specific entity.
+    members:
+      OriginalRepresentationRelationship:
```

Comment:
[Question] Does calling these `XRelationship` break our suggestion about using the namespace to qualify meaning?

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/71/commits/f231689ca43155a07d4b667e9efca0f8fe2fb4bc I dont think we can assume that all relationships would be in a `relationship` namespace (see the existing ones), as we only have one level. It felt useful to make code more readable/understandable, and packages easier to grok/autocomplete if we do include it.